exporter/exporterhelper/common_test.go

```diff
@@ -36,26 +38,17 @@ func TestWithSpanName(t *testing.T) {
 }
 
 func TestErrorToStatus(t *testing.T) {
-	if g, w := errToStatus(nil), okStatus; g != w {
-		t.Fatalf("Status: Want %v Got %v", w, g)
-	}
-
-	errorStatus := trace.Status{Code: trace.StatusCodeUnknown, Message: "my_error"}
-	if g, w := errToStatus(errors.New("my_error")), errorStatus; g != w {
-		t.Fatalf("Status: Want %v Got %v", w, g)
-	}
+	errorStatus := errToStatus(nil)
+	require.Equalf(t, okStatus, errorStatus, "Status: Want %v Got %v", okStatus, errorStatus)
```

Comment:
Use `require.Equal` and do not require the message, in general avoid the `f` functions. Apply this to the other locations, I think that the message is just worth if is something that requires some explanation.